astropy_helpers/extern/automodapi/automodapi.py

```diff
@@ -1,13 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """
-This sphinx extension adds a tools to simplify generating the API
-documentation for Astropy packages and affiliated packages.
-
-.. _automodapi:
-
-========================
-automodapi directive
```

Comment:
Why was this removed?  It seems like the header is useful in the generated docs...?